### DISCUSSION TITLE
Weird energies compared to FHI-aims

### DISCUSSION THREAD

**Discussion by EYErbil** [deepmodeling/deepmd-kit#4057] [Q&A]
Hello, I am comparing the calculations with aims and my graph .pb , 
this is my ML model : 
[relaxation.log](https://github.com/user-attachments/files/16626514/relaxation.log)
Here is FHI-aims DFT output:
[relaxation.log](https://github.com/user-attachments/files/16626521/relaxation.log)
Energies are really different, what I first thought is, I trained my model with CP2K outputs, so the energies should be Hartree, thus, I multiply the energy with 27, but still, it is 1/2 of what I get, any ideas?

**Comment by njzjz** [comment]
Are you sure CP2K and FHI-aims have the same output? They may have different energy shifts.
